Index.html

```diff
@@ -7,33 +7,44 @@
     <title>Portfolio of Nahid Raihan</title>
     <link rel="stylesheet" href="style.css">
   </head>
-  <body>
+  <body class="body-area">
     
-    <header class="menu-button">
-      <img src="media/Enabled.svg" alt="menu bar">
+    <header class="header-cover">
+      <img class="menu-button" src="media/Enabled.svg" alt="menu bar">
     </header>
+    <nav class="nav-bar">
+      <ul class="nav-bar-ul">
+        <a href="#"><li>Portoflio</li></a>
+        <a href="#"><li>About</li></a>
+        <a href="#"><li>Contact</li></a>
+      </ul>
+    </nav>
     <main class="main-container">
       <div class="main-text">
         <div class="main-text-blue">
-          <h1>Hey there. <br> I’m Nahid.</h1>
+          <p>Hey there. I’m Nahid.</p>
         </div>
         <div class="main-text-green">
-         <h2>I’m a software developer</h2>
+          <p>I’m a software developer</p>
         </div>
         <div class="intro-text">
           <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
           don’t hesitate to contact me.</p>
         </div>
 
         <ul class="social-media">
-         <li><a><img src="media/twitter.svg" alt="twiter lonk"></a></li>
-         <li><a></a><img src="media/Linkedin icon.svg" alt="linkedin link"></a></li>
-         <li><a><img src="media/medium.svg" alt="medium link"></a></li>
-         <li><a><img src="media/github.svg" alt="github link"></a></li>
-         <li><a><img src="media/angellist.svg" alt="angellint link"></a></li>
+         
+          <li><a><img src="media/github.svg" alt="github link"></a></li>
+          <li><a></a><img src="media/Linkedin icon.svg" alt="linkedin link"></a></li>
+          <li><a><img src="media/angellist.svg" alt="angellint link"></a></li>
+          <li><a><img src="media/twitter.svg" alt="twiter lonk"></a></li>
+          <li><a><img src="media/medium.svg" alt="medium link"></a></li>
+         
+         
         </ul>
-
-        <a class="disable-button"><img src="media/Disabled.svg" alt=""></a>
+        <div class="disable-img">
+          <a class="disable-button"><img src="media/Disabled.svg" alt=""></a>
+        </div>
  
       </div>
       <section class="work-container">
```

Comment:
- [ ] Kindly add some margin at the top to separate the first section from the projects section like the Figma template.
 - Your design:
 - 
<img width="1452" alt="image" src="https://user-images.githubusercontent.com/61919617/228854663-dcf800c6-fd5b-43bf-a978-d8ab9e635359.png">

 - Figma template:
 - 
<img width="703" alt="image" src="https://user-images.githubusercontent.com/61919617/228854786-21bdb213-2c98-4346-a23d-7b4b8b5bf78f.png">

Reply:
resolved